AdobeStockAsset/Model/Config.php

```diff
@@ -39,6 +42,6 @@ public function __construct(ScopeConfigInterface $scopeConfig)
      */
     public function isEnabled(): bool
     {
-        return (bool)$this->scopeConfig->getValue(self::XML_PATH_ENABLED);
+        return (bool)$this->scopeConfig->isSetFlag(self::XML_PATH_ENABLED);
```

Comment:
`isSetFlag` returns bool, no need to cast it.
```suggestion
        return $this->scopeConfig->isSetFlag(self::XML_PATH_ENABLED);
```